plugins/hypervisors/kvm/src/test/java/com/cloud/hypervisor/kvm/resource/LibvirtComputingResourceTest.java

```diff
@@ -416,25 +416,6 @@ public void testCreateDevicesDef() {
         verifyDevices(devicesDef, to);
     }
 
-    @Test
```

Comment:
@rhtyd 
code lgtm

is it necessary to remove this unit test ?

Reply:
Yes, as it was checking that for aarch64 scsi would always return; now it's not the case.

Reply:
good. just tested it can be fixed by adding ```to.setPlatformEmulator("");```

Reply:
@RohitYadavCloud 
I just had a look at the test again.  I think we should not remove the test.
we could apply the following change 
```
-        libvirtComputingResourceSpy._guestCpuArch = "aarch64";
+        to.setPlatformEmulator("Other PV Virtio-SCSI");
```
unit test will pass

Reply:
Agree with @weizhouapache on keeping the test with the needed adjustments.